spec/definitions/GlobalTimelineEvent.yaml

```diff
@@ -0,0 +1,33 @@
+type: object
+description: Global Timeline event.
+readOnly: true
+properties:
```

Comment:
I think we should keep `eventType` if we want to be able to filter by them.

Reply:
With these recent changes, I think that we can add a filter for `resourceType` instead. The point of having `eventType` was that each type would determine the schema of `data` attribute. Since now those are unified, we can probably put `eventType ` aside.

Reply:
After some discussions, we decided to bring it back (and keep the filter too).